scriptHelper.js

```diff
@@ -1,43 +1,127 @@
+            </ol>
+            <img src="${imageUrl}">`;
+
+  // Here is the HTML formatting for our mission target div.
+  // /*
+  //               <h2>Mission Destination</h2>
+  //               <ol>
+  //                   <li>Name: </li>
+  //                   <li>Diameter: </li>
+  //                   <li>Star: ${star}</li>
+  //                   <li>Distance from Earth: </li>
+  //                   <li>Number of Moons: </li>
+  //               </ol>
+  //               <img src="">
+  //  */
 }
 
 function validateInput(testInput) {
-   
+  if (testInput === '') {
+    return 'Empty';
+  } else if (isNaN(testInput)) {
+    return 'Not a Number';
+  } else if (!isNaN(testInput)) {
+    return 'Is a Number';
+  }
 }
 
 function formSubmission(document, list, pilot, copilot, fuelLevel, cargoLevel) {
-   
+  let pilotValidation = validateInput(pilot);
+  let copilotValidation = validateInput(copilot);
+  let fuelLevelValidation = validateInput(fuelLevel);
+  let cargoLevelValidation = validateInput(cargoLevel);
+  let pilotStatus = document.getElementById('pilotStatus');
+  let copilotStatus = document.getElementById('copilotStatus');
+  let fuelStatus = document.getElementById('fuelStatus');
+  let cargoStatus = document.getElementById('cargoStatus');
+  let launchStatus = document.getElementById('launchStatus');
+
+  if (
+    pilotValidation === 'Empty' ||
+    copilotValidation === 'Empty' ||
+    fuelLevelValidation === 'Empty' ||
+    cargoLevelValidation === 'Empty'
+  ) {
+    alert('All fields are required!');
+  } else if (
+    pilotValidation === 'Is a Number' ||
+    copilotValidation === 'Is a Number'
+  ) {
+    alert('Enter a valid name');
+  } else if (
+    fuelLevelValidation === 'Not a Number' ||
+    cargoLevelValidation === 'Not a Number'
+  ) {
+    alert('Enter a valid number');
+  }
+  pilotStatus.innerHTML = `Pilot ${pilot} is ready for launch`;
+  copilotStatus.innerHTML = `Co-pilot ${copilot} is ready for launch`;
+
+  if (Number(fuelLevel) < 10000) {
+    list.style.visibility = 'visible';
+    launchStatus.style.color = 'rgb(199, 37, 78)';
+    launchStatus.innerHTML = `Shuttle Not Ready for Launch`;
+    fuelStatus.innerHTML = `Fuel level too low for launch`;
+    cargoLevel.innerHTML = `Cargo mass too heavy for launch`;
+  }
+
+  if (Number(cargoLevel) > 10000) {
+    list.style.visibility = 'visible';
+    launchStatus.style.color = 'rgb(199, 37, 78)';
+    launchStatus.innerHTML = `Shuttle Not Ready for Launch`; 
+    fuelLevel.innerHTML = `Fuel level too low for launch`;
+    cargoStatus.innerHTML = `Cargo mass too heavy for launch`;
+  }
+
+  if (Number(cargoLevel) <= 10000 && Number(fuelLevel) >= 10000) {
+    list.style.visibility = 'visible';
+    launchStatus.style.color = 'rgb(65, 159, 106)';
+    launchStatus.innerHTML = `Shuttle is Ready for Launch`;
+    fuelLevel.innerHTML = `Fuel level high enough for launch`;
+    cargoLevel.innerHTML = `Cargo mass low enough for launch`;
+  }
```

Comment:
With the way the tests are written, it expects that after you put in either too low of fuel, or too high of cargo weight, you then enter correct weights/fuel levels. And when you do that, all of the prompts should reset to the original values. So you'd need another if that checks if the cargoLevel is high enough, and cargo weight is low enough. And if so, the statuses all reset. The logic for everything else looks perfect though!